<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingWidgetAbsValueAtOrigin</class>
 <widget class="QWidget" name="SettingWidgetAbsValueAtOrigin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_12">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_16">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>单
圈
数
据
设
置</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_single1">
             <property name="font">
              <font>
               <pointsize>24</pointsize>
              </font>
             </property>
             <property name="text">
              <string>1# 轴:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_singleAbsValue_1">
             <property name="font">
              <font>
               <pointsize>24</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_single2">
             <property name="font">
              <font>
               <pointsize>24</pointsize>
              </font>
             </property>
             <property name="text">
              <string>2# 轴:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_singleAbsValue_2">
             <property name="font">
              <font>
               <pointsize>24</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_single3">
             <property name="font">
              <font>
               <pointsize>24</pointsize>
              </font>
             </property>
             <property name="text">
              <string>3# 轴:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_singleAbsValue_3">
             <property name="font">
              <font>
               <pointsize>24</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_single4">
             <property name="font">
              <font>
               <pointsize>24</pointsize>
              </font>
             </property>
             <property name="text">
              <string>4# 轴:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_singleAbsValue_4">
             <property name="font">
              <font>
               <pointsize>24</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_single5">
             <property name="font">
              <font>
               <pointsize>24</pointsize>
              </font>
             </property>
             <property name="text">
              <string>5# 轴:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_singleAbsValue_5">
             <property name="font">
              <font>
               <pointsize>24</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_single6">
             <property name="font">
              <font>
               <pointsize>24</pointsize>
              </font>
             </property>
             <property name="text">
              <string>6# 轴:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_singleAbsValue_6">
             <property name="font">
              <font>
               <pointsize>24</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>多
圈
数
据
设
置</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_multi1">
             <property name="font">
              <font>
               <pointsize>24</pointsize>
              </font>
             </property>
             <property name="text">
              <string>1# 轴:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_multiAbsValue_1">
             <property name="font">
              <font>
               <pointsize>24</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_multi2">
             <property name="font">
              <font>
               <pointsize>24</pointsize>
              </font>
             </property>
             <property name="text">
              <string>2# 轴:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_multiAbsValue_2">
             <property name="font">
              <font>
               <pointsize>24</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="label_multi3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>24</pointsize>
              </font>
             </property>
             <property name="text">
              <string>3# 轴:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_multiAbsValue_3">
             <property name="font">
              <font>
               <pointsize>24</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLabel" name="label_multi4">
             <property name="font">
              <font>
               <pointsize>24</pointsize>
              </font>
             </property>
             <property name="text">
              <string>4# 轴:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_multiAbsValue_4">
             <property name="font">
              <font>
               <pointsize>24</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="QLabel" name="label_multi5">
             <property name="font">
              <font>
               <pointsize>24</pointsize>
              </font>
             </property>
             <property name="text">
              <string>5# 轴:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_multiAbsValue_5">
             <property name="font">
              <font>
               <pointsize>24</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_14">
           <item>
            <widget class="QLabel" name="label_multi6">
             <property name="font">
              <font>
               <pointsize>24</pointsize>
              </font>
             </property>
             <property name="text">
              <string>6# 轴:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_multiAbsValue_6">
             <property name="font">
              <font>
               <pointsize>24</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_singleAbsValue_1</tabstop>
  <tabstop>lineEdit_multiAbsValue_1</tabstop>
  <tabstop>lineEdit_singleAbsValue_2</tabstop>
  <tabstop>lineEdit_multiAbsValue_2</tabstop>
  <tabstop>lineEdit_singleAbsValue_3</tabstop>
  <tabstop>lineEdit_multiAbsValue_3</tabstop>
  <tabstop>lineEdit_singleAbsValue_4</tabstop>
  <tabstop>lineEdit_multiAbsValue_4</tabstop>
  <tabstop>lineEdit_singleAbsValue_5</tabstop>
  <tabstop>lineEdit_multiAbsValue_5</tabstop>
  <tabstop>lineEdit_singleAbsValue_6</tabstop>
  <tabstop>lineEdit_multiAbsValue_6</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
